eox_nelp/stats/templates/tenant_stats.html

```diff
@@ -8,11 +8,11 @@
 <body>
     <div id="tenant-stats"></div>
     <script>
-      var showVideos = "true" === "${videos}";
-      var showCourses = "true" === "${courses}";
-      var showProblems = "true" === "${problems}";
-      var showLearners = "true" === "${learners}";
-      var showInstructors = "true" === "${instructors}";
+      var showVideos = "false" === "${hideVideos}";
+      var showCourses = "false" === "${hideCourses}";
+      var showProblems = "false" === "${hideProblems}";
+      var showLearners = "false" === "${hideLearners}";
```

Comment:
Here is that thing where we have to consider changing the names and matching connection with component, why if the component has ShowCourses we have to connect it with hideCourses? This could be confusing.
[😶‍🌫️](https://emojipedia.org/face-in-clouds/)

Reply:
this comment is weird since you removed the thing that you are proposing

Reply:
Actually, I changed it to true in my first approach. Then due to my confusion about snake_case and camel_case, I changed to videos, courses, problems.learners.. they all fit snake and camel case

Reply:
yes, I checked again your changes and you didn't  proposed any change in the way how the api is consumed, you only changed the default values to show everything  but using confusing names